src/components/media/MediasSelectModal.jsx

```diff
@@ -203,9 +199,5 @@ export default MediasSelectModal
 
 MediasSelectModal.propTypes = {
   onClose: PropTypes.func.isRequired,
-  siteName: PropTypes.string.isRequired,
   onMediaSelect: PropTypes.func.isRequired,
-  type: PropTypes.oneOf(["files", "images"]).isRequired,
-  readFileToStageUpload: PropTypes.func.isRequired,
-  setUploadPath: PropTypes.func.isRequired,
 }
```

Comment:
Why are the proptypes here removed?

Reply:
they're never used

Reply:
Ah i think I saw wrongly - there were missing proptypes for this component but not for the params removed